packages/shopify-app-session-storage-postgresql/src/postgresql.ts

```diff
@@ -43,6 +45,9 @@ export class PostgreSQLSessionStorage implements SessionStorage {
     }
     this.options = {...defaultPostgreSQLSessionStorageOptions, ...opts};
     this.ready = this.init();
+
+    const config = parse(dbUrl.toString());
+    this.database = config.database || 'postgres';
```

Comment:
I would suggest to use the right side of your assignment straight where `this.database` is used L145. I think it increases readability and reduces ever so slightly the cognitive load.